qlora.py

```diff
@@ -465,6 +466,19 @@ def extract_alpaca_dataset(example):
         prompt_format = PROMPT_DICT["prompt_no_input"]
     return {'input': prompt_format.format(**example)}
 
+def local_dataset(dataset_name):
+    if dataset_name.endswith('.json'):
```

Comment:
Perhaps check for .jsonl as well as .json

Reply:
Good point, added